### ISSUE TITLE
Offline Plotly - saving notebook causes JSON error

### ISSUE THREAD

**Issue by gregmoille** [jupyterlab/jupyter-renderers#158]
Hi all, 

Using the Plotly extension for Jupiter lab works fine but when I save the notebook, a JSON error happen stating that is invalid.

The example I ran in a Jupiter lab notebook: 
```python
from plotly.offline import download_plotlyjs, init_notebook_mode, plot, iplot
init_notebook_mode(connected=True)
iplot([{"x": [1, 2, 3], "y": [3, 1, 6]}])
```

Here is the error when saving the notebook

```
Notebook JSON is invalid: {'x': [1, 2, 3], 'y': [3, 1, 6], 'type': 'scatter', 'uid': '22e994e2-b776-11e8-800b-8c8590492a0e'} is not valid under any of the given schemas

Failed validating 'oneOf' in display_data['properties']['data']['additionalProperties']:

On instance['cells'][0]['outputs'][1]['data']['0']:
{'type': 'scatter',
     'uid': '22e994e2-b776-11e8-800b-8c8590492a0e',
     'x': [1, 2, 3],
     'y': [3, 1, 6]}
```

The saved notebook still works, but the Plotly plots are now still images (I guess some type of png).

This issue is not reproduced using a _normal_ Jupyter notebook. Also opening a saved notebook with the _normal_ Jupyter works (i.e. Plotly plots are interactive and not png), but saving this same notebook in the Jupyter lab env causes the issue.

Thanks !

**Comment by jonmmease** [comment]
Fixed by #159 so this can be closed. cc @gregmoille @blink1073

**Comment by blink1073** [comment]
Thanks for the heads up!
